qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_model.py

```diff
@@ -783,14 +814,27 @@ def _evaluate_plot_guess(self, plot_guess: bool) -> str:
 
     def _evaluate_function(self, fit_function: IFunction, output_workspace: str) -> str:
         """Evaluate the plot guess fit function and returns the name of the resulting guess workspace."""
+        if self.fitting_context.plot_guess_type == 'x from fit range':
```

Comment:
I think it would be good to create three constant variables for 'x from fit range', 'Uniform points across data range' and 'Custom x range' and but them in the `basic_fitting_context` here:
https://github.com/mantidproject/mantid/blob/cffb8050f43a522a2ff059a7834b0e4f59aa1075/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/basic_fitting_context.py#L10

Then import them to this file and do something like this for each if clause
```
if self.fitting_context.plot_guess_type == PLOT_GUESS_FIT_RANGE:
```